datalad/distributed/tests/test_ria_basics.py

```diff
@@ -392,6 +393,13 @@ def _test_gitannex(host, store, dspath):
     store = Path(store)
 
     ds = Dataset(dspath).create()
+
+    if ds.repo.is_managed_branch():
+        # git-annex-testremote is way too slow on crippled FS.
+        # Use is_managed_branch() as a proxy and skip only here
+        # instead of in a decorator
+        raise SkipTest("Test tto slow on crippled FS")
```

Comment:
"too" typo

Reply:
Guess that comment is a `"too" type objection`. So - "tto". :-P

Force-push in a sec